_analyzers/stop.md

```diff
@@ -0,0 +1,147 @@
+---
+layout: default
+title: Stop analyzer
+nav_order: 70
+---
+
+# Stop analyzer
+
+The `stop` analyzer removes a predefined list of stopwords. This analyzer consists of a `lowercase` tokenizer and a `stop` token filter.
+
+## Parameters
+
+You can configure a `stop` analyzer with the following parameters.
+
+Parameter | Required/Optional | Data type | Description
+:--- | :--- | :--- | :--- 
+`stopwords` | Optional | String or list of strings | A string specifying a predefined list of stopwords (such as `_english_`) or an array specifying a custom list of stopwords. Default is `_english_`.
+`stopwords_path` | Optional | String | The path (absolute or relative to the config directory) to the file containing a list of stop words.
```

Comment:
I think it would be good to add an example where we would be using stopwords_path and format it expects

```
is
a
or
and
```